prime-angular-frontend/src/assets/pill-1.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="1272px" height="608px" viewBox="0 0 1272 608" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 61.2 (89653) - https://sketch.com -->
+    <title>pill 1</title>
+    <desc>Created with Sketch.</desc>
```

Comment:
Should this be here?

Reply:
Can pull this out of the svg, as I assume we don't want this information there.